frontend/src/views/EmulatorJS/Player.vue

```diff
@@ -101,7 +101,11 @@ window.EJS_color = "#A453FF";
 window.EJS_alignStartButton = "center";
 window.EJS_startOnLoaded = true;
 window.EJS_backgroundImage = "/assets/emulatorjs/loading_black.png";
-window.EJS_defaultOptions = { "save-state-location": "browser" };
+window.EJS_defaultOptions = {
+  "save-state-location": "browser",
+  rewindEnabled: "enabled",
+  fastForward: "enabled",
```

Comment:
this doesnt just enable the option to fast forward, it runs the game fast forward. needs to be reverted. rewind enabled can stay though, that one works.

Reply:
Oh damn, good catch and removed